src/Receipts/ReceiptResponse.php

```diff
@@ -63,19 +63,18 @@ public function __construct(array $attributes)
         }
 
         $this->environment = $attributes['environment'];
-        $this->latestReceipt = $attributes['latest_receipt'];
+        $this->latestReceipt = $attributes['latest_receipt'] ?? '';
```

Comment:
I'd use null here to better reflect the absence of value. An empty string is an invalid value for a receipt

Reply:
Thanks for your feedback. I wasn't 100% sure. I've updated the default value.

Reply:
Appreciate your PR, thanks!